### PR TITLE
Adding support for model column :time

### PR DESCRIPTION
**Pull request by osiro**
Now you can do:

```
shift:
  attributes:
    start_at:
      type: "time"
      validates:
        presence: true
    end_at:
      type: "time"
      validates:
        presence: true
```

### CODE REVIEW

File: lib/model_configuration/attribute/factory_declaration.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def to_s
 
   def data_for_attribute
     case attribute.type
-    when "datetime", "date"
+    when "datetime", "date", "time"
```
Comment (jordanmaguire) [inline lib/model_configuration/attribute/factory_declaration.rb]: Should be a corresponding test for this.

Comment (osiro) [inline lib/model_configuration/attribute/factory_declaration.rb]: Done :)

Comment (osiro) [inline lib/model_configuration/attribute/factory_declaration.rb]: Actually... hold on..

**Comment by jordanmaguire** [comment]
:+1:

File: lib/model_configuration/attribute/factory_declaration.rb (hunk 1 above)
Comment (osiro) [inline lib/model_configuration/attribute/factory_declaration.rb]: Now yeah! Done

**Pull request merged**